docs/getting_started/AppDatamodelUsageWorkflow.md

```diff
@@ -0,0 +1,142 @@
+## App Datamodel Usage Workflow
+
+This workflow will walk you through the steps to interact and work with your applocal datamodel.
+
+* [Pre-requisites](AppDatamodelUsageWorkflow.md#pre-requisites)
+
+* [Usage examples with Nexus Tools](AppDatamodelUsageWorkflow.md#usage-nexus-tools)
+
+
+## Pre-requisites
+1. This workflow requires Datamodel to be initialised and installed before proceeding to further steps. Please follow the below link to configure datamodel
+
+    #### [Datamodel Workflow](DatamodelWorkflow.md)
+
+<!--
+
+1. This workflow requires the App should be initialised and configured before proceeding to further steps. Please follow the below link to configure the App
+
+    #### [App Workflow](AppWorkflow.md)
+-->
+
+<!-- nexus-specific exports
+```
+# store the current directory before we `cd` into the app dir
+export DOCS_INTERNAL_DIR=$PWD/docs/_internal
+```
+-->
+
+
+## Usage examples with Nexus Tools
+
+```
+# Portforward the services Nexus API Gw and GraphQL to access the datamodel objects
+kubectl port-forward svc/nexus-api-gw 5000:80 -n default &
+```
+
+###  Create Leader Object via REST API
+
+**GraphQL query to verify that leader doesn't exist**
+
```

Comment:
can u insert copyable link to graphql server, so user doesn't have to retype from image?

Reply:
Will do it in another PR. Added a ticket to track : https://jira.eng.vmware.com/browse/NPT-640